lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -489,19 +514,16 @@ private void tryLocks(LockClient client, LockRequest request, BlockingMode block
     }
 
     @VisibleForTesting
-    @SuppressWarnings("Slf4jConstantLogMessage")
     protected void logSlowLockAcquisition(String lockId, LockClient currentHolder, long durationMillis) {
-        String slowLockLogMessage = "Blocked for {} ms to acquire lock {} {}.";
+        final String slowLockLogMessage = "Blocked for {} ms to acquire lock {} {}.";
```

Comment:
I might keep the log message construction in each of the branches below to save having to construct the collection and convert to the array when most of the time we don't need to do these things.